src/analysis/retail/shaman/restoration/normalizers/CastLinkNormalizer.ts

```diff
@@ -394,8 +409,16 @@ export function isLivelyTotemsChainHeal(event: HealEvent) {
   return HasRelatedEvent(event, LIVELY_TOTEMS_CHAIN_HEAL);
 }
 
-export function isReactivityHeal(event: HealEvent) {
-  return HasRelatedEvent(event, REACTIVITY);
+export function isSplitstreamHeal(event: HealEvent) {
+  return HasRelatedEvent(event, SPLITSTREAM);
+}
+
+export function earthlivingApplication(event: ApplyBuffEvent | RefreshBuffEvent) {
```

Comment:
This seems like it should be returning a singular event via `GetRelatedEvent` which would then allow you to check the event itself instead of first-indexing in `EarthlivingWeapon.tsx => onApplyEarthliving`